contracts/sol6/mock/ReentrancyAttack.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.6.6;
```

Comment:
Seems this contract isn't used here.
can remove.

Reply:
Still used in the tests somehow.